<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>1149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbAction">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>open path</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtParam">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>171</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>environment </string>
    </property>
    <widget class="QRadioButton" name="rbFramework">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>95</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Framework</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbMyTest">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>95</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>MyTest</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbStyle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Style</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbAutoRecord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>AutoRecrod</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbManualRecord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>ManualRecord</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbVideoDb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>VideoDb</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbAction">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox QAbstractItemView{
    font-size: 18px;
    background-color: rgb(245,246,247);
	selection-color: rgb(255,255,255); /* 文本选中时字体颜色 */
}</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtOutput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>760</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>740</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Param</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAction">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogAction">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LogAction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtMsg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>821</width>
      <height>891</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClearMsg">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTest">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHL">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message List:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1314</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
